ucm2/sof-hda-dsp/sof-hda-dsp.conf

```diff
@@ -5,7 +5,27 @@ SectionUseCase."HiFi" {
 	Comment "Play HiFi quality Music"
 }
 
-SectionUseCase."Hdmi" {
-	File "Hdmi.conf"
-	Comment "Play to HDMI/DP"
+SectionUseCase."Hdmi1" {
+	File "Hdmi1.conf"
+	Comment "Play to HDMI1/DP1"
 }
+
+SectionUseCase."Hdmi2" {
+	File "Hdmi2.conf"
+	Comment "Play to HDMI2/DP2"
+}
+
+SectionUseCase."Hdmi3" {
+	File "Hdmi3.conf"
+	Comment "Play to HDMI3/DP3"
+}
+
+SectionDefaults [
+	cdev "hw:sofsklhdacard"
+	cset "name='Master Playback Switch' on"
```

Comment:
The Master volume is just a soft-volume attenuator in this case. It should be set to 100%. Decrease the analog volume control in hw (Speaker Playback Volume, Headphone Playback Volume).

Reply:
ok fine by me, I've just taken previous "safe" values found, let me check this doesn't distort the audio or anything.

Reply:
Thinking more: We should not probably touch those values. I implemented the master control for PA in https://gitlab.freedesktop.org/pulseaudio/pulseaudio/merge_requests/213 . It seems working nicely here.

The 'Capture Switch' and 'Auto-Mute Mode' settings is fine (although I think that the auto mute settings is available only for some HDA codecs).

Reply:
If I recall correctly this was more for "first boot" situation where pulseaudio doesn't have anything in its databases. So if the PR implements the element chaining that raising the headphone/speaker volume also opens Master, then removing it would be fine.